docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2916 @@
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
 
-Review important information about the {kib} 8.0.0 releases.
+Review important information about the {kib} 8.6.0 release.
+
+* <<release-notes-8.6.0>>
+//* <<release-notes-8.5.2>>
+//* <<release-notes-8.5.1>>
+//* <<release-notes-8.5.0>>
+//* <<release-notes-8.4.3>>
+//* <<release-notes-8.4.2>>
+//* <<release-notes-8.4.1>>
+//* <<release-notes-8.4.0>>
+//* <<release-notes-8.3.3>>
+//* <<release-notes-8.3.2>>
+//* <<release-notes-8.3.1>>
+//* <<release-notes-8.3.0>>
+//* <<release-notes-8.2.3>>
+//* <<release-notes-8.2.2>>
+//* <<release-notes-8.2.1>>
+//* <<release-notes-8.2.0>>
+//* <<release-notes-8.1.3>>
+//* <<release-notes-8.1.2>>
+//* <<release-notes-8.1.1>>
+//* <<release-notes-8.1.0>>
+//* <<release-notes-8.0.1>>
+//* <<release-notes-8.0.0>>
+//* <<release-notes-8.0.0-rc2>>
+//* <<release-notes-8.0.0-rc1>>
+//* <<release-notes-8.0.0-beta1>>
+//* <<release-notes-8.0.0-alpha2>>
+//* <<release-notes-8.0.0-alpha1>>
 
-* <<release-notes-8.0.0>>
-* <<release-notes-8.0.0-rc2>>
-* <<release-notes-8.0.0-rc1>>
-* <<release-notes-8.0.0-beta1>>
-* <<release-notes-8.0.0-alpha2>>
-* <<release-notes-8.0.0-alpha1>>
+--
+
+[[release-notes-8.6.0]]
+== {kib} 8.6.0
+
+coming::[8.6.0]
+
+Review the following information about the {kib} 8.6.0 release.
+
+[float]
+[[breaking-changes-8.6.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.6.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+[discrete]
+[[breaking-143081]]
+.Changes the `histogram:maxBars` default setting
+[%collapsible]
+====
+*Details* +
+To configure higher resolution data histogram aggregations without changing the *Advanced Settings*, the default histogram:maxBars setting is now 1000 instead of 100. For more information, refer to {kibana-pull}143081[#143081].
+
+*Impact* +
+For each <<xpack-spaces,space>>, complete the following to change *histogram:maxBars* to the previous default setting:
```

Comment:
This link causes a doc build error when it's re-used in the Installation and Upgrade Guide, so it needs to be an external link:
```suggestion
For each {kibana-ref}/xpack-spaces.html[space], complete the following to change *histogram:maxBars* to the previous default setting:
```